python-squid/src/main/java/org/sonar/python/metrics/FileLinesVisitor.java

```diff
@@ -24,13 +24,16 @@
 import com.sonar.sslr.api.GenericTokenType;
 import com.sonar.sslr.api.Token;
 import com.sonar.sslr.api.Trivia;
+import java.util.Collection;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Collection'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-179&issues=AWD6HF7bEFfjxkQcPpim&open=AWD6HF7bEFfjxkQcPpim)

[ISSUE_KEY]: AWD6HF7bEFfjxkQcPpim